Kernel/Arch/riscv64/Processor.h

```diff
@@ -12,6 +12,7 @@
 
 #include <Kernel/API/POSIX/errno.h>
 #include <Kernel/Arch/DeferredCallPool.h>
+#include <Kernel/Arch/Processor.h>
```

Comment:
Were these headers included transitively? Or why are they necessary now?

Reply:
Essentially all users of this header also include that one, but my clangd was complaining that it did not know `ProcessorBase`, so here we are